internal/apps/dop/providers/issue/dao/core.go

```diff
@@ -161,6 +161,10 @@ func (client *DBClient) UpdateIssues(requirementID uint64, fields map[string]int
 		Updates(fields).Error
 }
 
+func (client *DBClient) BatchUpdateIssueIterationID(ids []uint64, iterationID int64) error {
+	return client.Model(Issue{}).Where("deleted = 0").Where("id IN (?)", ids).Updates(Issue{IterationID: iterationID}).Error
```

Comment:
- you'd better check ids firstly
- use map in `Updates` is better (bool are easily overlooked)

Reply:
ok,thanks